pkg/arm/resources.go

```diff
@@ -149,6 +152,7 @@ func vnet(cs *api.OpenShiftManagedCluster) *network.VirtualNetwork {
 				},
 			},
 		},
+		ID:       to.StringPtr(resourceID(cs.Properties.AzProfile.SubscriptionID, cs.Properties.AzProfile.ResourceGroup, "Microsoft.Network/virtualNetworks", vnetName)),
```

Comment:
this shouldn't/needn't be here.  Azure is assigning the ID; we're reporting it back to the user.  We're not changing how we request it.

Reply:
Reverted change to the ARM file.